File: SeirpinskiGasketAndroid/app/src/main/java/com/example/mukesh/seirpinskigasketandroid/MainActivity.java
package com.example.mukesh.seirpinskigasketandroid;

import android.opengl.GLSurfaceView;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.MotionEvent;
import java.lang.Runnable;

public class MainActivity extends AppCompatActivity {
    private MyGLSurfaceView mGLView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mGLView = new MyGLSurfaceView(this);
        setContentView(mGLView);
    }
}

File: DeepFekuGram/app/src/main/java/com/example/deepfekugram/MainActivity.kt
package com.example.deepfekugram  // <- replace with your actual package

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.deepfekugram.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}
